catalog/core/catalog-core-metacardgroomerplugin/src/test/java/ddf/catalog/plugin/groomer/TestMetacardGroomerPlugin.java

```diff
@@ -63,23 +69,21 @@
  * @author ddf.isgs@lmco.com
  */
 public class TestMetacardGroomerPlugin {
```

Comment:
I might just be overlooking it, but I don't see a test in here to assert that an update of a metacard that has attribute for `CORE.METACARD_CREATED` get a created date attr.